inference-engine/src/gna_plugin/gna_device.cpp

```diff
@@ -217,7 +213,12 @@ uint32_t GNADeviceHelper::createRequestConfig(const uint32_t model_id) {
     // (bit exactly) as on the selected GNA execution target generation.
     // See the GNA Plugin's GNA_EXEC_TARGET config option description.
     if (swExactMode) {
-        const auto consistentDevice = getTargetDevice(true);
+        Gna2DeviceVersion consistentDevice;
+        if (useDeviceEmbeddedExport) {
+            consistentDevice = Gna2DeviceVersionEmbedded1_0;
+        } else {
+            consistentDevice = getTargetDevice(true);
+        }
         status = Gna2RequestConfigEnableHardwareConsistency(reqConfId, consistentDevice);
         checkGna2Status(status, "Gna2RequestConfigEnableHardwareConsistency(" + std::to_string(static_cast<long>(consistentDevice)) + ")");
```

Comment:
We should remove `Gna2RequestConfigEnableHardwareConsistency` calls at this function is now deprecated!